Source/WebCore/platform/audio/SharedAudioDestination.cpp

```diff
@@ -61,31 +61,31 @@ class SharedAudioDestinationAdapter : public ThreadSafeRefCountedAndCanMakeThrea
     void isPlayingDidChange() final { }
 
     void configureRenderThread(CompletionHandler<void(bool)>&&);
+    void callAllConfigurationHandlers(bool);
 
     Ref<AudioDestination> protectedDestination() { return m_destination; }
     Ref<AudioBus> protectedWorkBus() { return m_workBus; }
-    Ref<WorkQueue> protectedConfigurationQueue() { return m_configurationQueue; }
 
     unsigned m_numberOfOutputChannels;
     float m_sampleRate;
 
     Ref<AudioDestination> m_destination;
     Ref<AudioBus> m_workBus;
-    Ref<WorkQueue> m_configurationQueue;
 
     bool m_started { false };
 
     Lock m_renderLock;
-    Semaphore m_configurationSemaphore;
 
-    // Only accessed on m_configurationQueue:
-    Vector<RefPtr<SharedAudioDestination>> m_renderers;
+    using RenderVector = Vector<RefPtr<SharedAudioDestination>>;
+    RenderVector m_renderers;
```

Comment:
it is unclear at a glance that this is only ever used on the main thread. a `WTF_GUARDED_BY_CAPABILITY(mainThread)` would increase readability (and the associated assertIsCurrent(isMainThread())